internal/api/v1/deploy/deploy.go

```diff
@@ -60,7 +61,7 @@ func DeployApp(ctx context.Context, cluster *kubernetes.Cluster, app models.AppR
 		Configurations: appObj.Configuration.Configurations,
 		Instances:      *appObj.Configuration.Instances,
 		ImageURL:       imageURL,
-		Username:       username,
+		Username:       user.ID,
```

Comment:
Hmm, changing `Username` here would require us to change the application chart too. I wonder if it's worth it.